README.md

```diff
@@ -51,6 +51,7 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 Name     | Description | OS
 ---------|-------------|----
+bcache | Exposes bcache statistics | Linux
```

Comment:
The code does handle systems without bcache gracefully. It will not return an error if no bcache files are found. The error message "failed to retrieve bcache stats: %v" is only encountered when reading stat files that do exist fails. It behaves exactly like xfs_linux.go. I just kept it disabled by default to minimize the potential impact of a new collector.